src/python_framework_v01/nhd_network_utilities_v01.py

```diff
@@ -636,6 +636,12 @@ def set_supernetwork_data(
             "driver_string": "NetCDF",
             "layer_string": 0,
             "waterbody_parameter_file_type": "Level_Pool",
+            "ql_input_folder": r"/home/APD/inland_hydraulics/wrf-hydro-run/OUTPUTS",
+            "ql_files_tail": "/*.CHRTOUT_DOMAIN1",
+            "time_string": "2020-03-19_18:00_DOMAIN1",
+            "channel_initial_states_file": r"/home/APD/inland_hydraulics/wrf-hydro-run/restart/HYDRO_RST.",
+            "initial_states_waterbody_ID_crosswalk_file": r"/home/APD/inland_hydraulics/wrf-hydro-run/DOMAIN/waterbody_subset_ID_crosswalk.csv",
+            "routelink_subset": r"/home/APD/inland_hydraulics/wrf-hydro-run/DOMAIN/routeLink_subset.nc",
```

Comment:
For these default networks, we probably don't want to hard code these paths. 
Instead, 
There should be logic when the paths could be used to either accept (in order of precedence)
1. a json custom file
1. a command-line argument, which would over-ride the .json (or print a warning for a conflict?)
1. a default value (such as a default constant qlat, or a 0/min value for initial flow/depth.)

Eventually, we would probably want to revisit any defaults and make sure we don't really want to throw an error if a value is not received, but for now, it will help.